server/services/healthCheck.ts

```diff
@@ -50,6 +50,7 @@ function gatherCheckInfo(aggregateStatus: Record<string, unknown>, currentStatus
 
 const apiChecks = [
   service('hmppsAuth', `${config.apis.hmppsAuth.url}/health/ping`, config.apis.hmppsAuth.agent),
+  service('manageUsersApi', `${config.apis.manageUsersApi.url}/health/ping`, config.apis.manageUsersApi.agent),
   ...(config.apis.tokenVerification.enabled
```

Comment:
Do you need `nomisUserRolesApi` in here?

Reply:
Yep - good spot! Added all the other APIs we use too as we only had the default ones from the template here.